doc/system-telemetry/process-docker-stats.md

```diff
@@ -0,0 +1,95 @@
+# Process and docker stats availability via telemetry agent
+
+## Revision
+
+| Rev | Date     | Author      | Change Description |
+|:---:|:--------:|:-----------:|--------------------|
+| 0.1 | 09/12/19 | Pradnya Mohite | Initial version    |
+
+## Scope
+-Enable sonic streaming telemetry agent to send Process and docker stats data
+
+### Enable sonic streaming telemetry agent to send Process and docker stats data
+
+##### Part 1 
+For 1st part, Daemon code will be added under sonic-buildimage/files/image_config.  A Daemon will start when OS starts. At every 2 min interval it will do following:  
+Delete all entries for Process and Docker stats from state db  
+Update Process and Docker stats data to state-DB.  
+Update last update time for process and Docker stats.  
+
+Details of CLI and state-DB given below. 
+
+##### Part 2
+Verify that from state-DB data is available via telemetry agent
+
+##### CLI output  and corresponding structure in state-DB for process and docker stats
+
+###### Process stats
+
+$ ps -eo uid,pid,ppid,%mem,%cpu,stime,tty,time,cmd
```

Comment:
Please place all console-based input/output in code blocks (three backticks) for better formatting

Reply:
Updated